app/javascript/angular/code/filtersUtils.js

```diff
@@ -95,3 +95,13 @@ const updateTooltipContent = (link, tooltip) => {
     window.setTimeout(tooltip.hide, 1000);
   });
 };
+
```

Comment:
I guess all of this file is untested, it could be a good exercise to see if there's any pure testable function to be extracted (in the same file or not) and test them